README.adoc

```diff
@@ -176,9 +176,7 @@ To learn how to configure the file, see https://openliberty.io/guides/microprofi
 Check that the `system` service availability returns a `200` response code.
 
 The `@Inject` annotation is used to initialize the `InventoryConfig` object. This object allows
-you to read the property in the `resources/CustomConfigSource.json` file. Learn more about the
-`@Inject` annotation from
-https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices].
+you to read the property in the `resources/CustomConfigSource.json` file.
```

Comment:
What about, "Use this object to read the property in the `resources/CustomConfigSource.json` file"?